pumpkin/src/entity/living.rs

```diff
@@ -72,6 +97,38 @@ impl LivingEntity {
         amount > 0.0
     }
 
+    pub async fn update_fall_distance(&self, dont_damage: bool) {
+        let y = self.entity.pos.load().y;
+        let last_y = self.entity.last_pos.load().y;
```

Comment:
You moved `last_pos` to `entity` to calculate the fall distance, Lets move `last_pos` to `living_entity` instead

Reply:
sounds like a good idea.
I think there was some reason i put it in `entity` instead of `living_entity`. But it not needed now and neither can i remember the reason.